crawl-ref/source/dat/descript/status.txt

```diff
@@ -333,6 +333,36 @@ You are silenced. This makes reading scrolls, casting spells, invoking divine
 abilities, or shouting impossible. The oppressive, unnatural effect greatly
 hampers your stealth.
 %%%%
+Engulf status
+
+{{
+    desc = "You are engulfed in water"
+
+    if you.race() == "Octopode"
```

Comment:
Consider using `if you.race():match("Octopode|Merfolk|Barachi") ...`; it can be clearer, and the current indentation isn't very nice.

Reply:
Sure thanks, that's the method I was looking for :)